alpha/experience-platform-extension/response-handling.md

```diff
@@ -1,16 +1,26 @@
 # Server response handling
 
-For each Experience Platform event that is sent through the Experience Platform Mobile extension, you can have no, or up to multiple, event handles returned from the Adobe solutions that you enabled. The event handle might contain information, such as personalization data, that can be used in the mobile application views, user profile updates, and so on.
+For each Experience Platform event that is sent through the Experience Platform Mobile extension, you can have zero or more event handles returned from the Adobe solutions that you enabled. The event handle might contain information, such as personalization data, that can be used in the mobile application views, user profile updates, and so on.
 
 ## Response handling
 
 Each event handle that is returned by the server is handled by the Experience Platform extension and is dispatched as a new mobile event to the extensions that are listening for it.
 
-If you integrated the Griffon extension in your mobile application, to display the events in your session, go to <https://griffon.adobe.com/> and in **Search Events**, type `Response event`.
+If you integrated the [Project Griffon extension](https://aep-sdks.gitbook.io/docs/beta/project-griffon) in your mobile application, to display the events in your session, go to <https://griffon.adobe.com/> and in **Search Events**, type `Response event`.
 
 ## Error handling
 
-If a server-side error occurs, the Experience Platform extension logs an error/warning message to logcat with the details. This message is dispatched as a new error response event for the mobile extensions to handle.
+If a server-side error occurs, the Experience Platform extension logs an error/warning message with the error or warning details, like in the following example:
```

Comment:
Please make the last part of the sentence separate like this:
"If a server-side error occurs, the Experience Platform extension logs an error/warning message with the error or warning details. For example:"

Reply:
@rekrishn implemented